app/utilities/account_functions.py

```diff
@@ -6,7 +6,8 @@
 
 from app.utilities.encryption import generate_password_hash, \
     check_password_hash, generate_secret_token, validate_password_format
-from app.utilities.mailing import send_verification_mail
+from app.utilities.mailing import \
+    send_verification_mail, send_forgot_password_mail
 
 
 async def get_account(email: str):
```

Comment:
If you don't want all the data in the database, better use MongoDB's `filter` keyword with `find_one` -- that's more performant as you'll simply not load those fields from the database.

Reply:
Yes! thank you :)